Makefile

```diff
@@ -115,11 +110,15 @@ vet: ## Run go vet against code.
 test: manifests generate fmt vet setup-envtest ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./... -coverprofile cover.out
 
+.PHONE: prepare-release
+prepare-release:
```

Comment:
can you please remove this for now (including the script) and do it in a separate PR that will introduce the release process?

Reply:
Done